data/sql/updates/pending_db_world/Update_quest_6002.sql

```diff
@@ -0,0 +1,23 @@
+(6002, 2, -1, 10, -263, 0, 0, 0, 0, 0, 0, 0, 5, 0, 0, 132300, 0, 19179, 0, 0, 15710, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 609, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 32, 'Body and Heart', 'Use the Cenarion Lunardust on the Moonkin Stone between Mulgore and the Barrens to bring forth Lunaclaw.  From there, you must face Lunaclaw and earn the strength of body and heart it possesses.$B$BSpeak with Turak Runetotem in Thunder Bluff when you are done.', 'On land west of Taurajo in the Barrens - just before it turns into Mulgore - lies a Moonkin Stone.  The moonkin are special beasts, not native to these parts.  Lunaclaw, the stone\'s defender, will face you when Cenarion Lunardust is applied to it.  Lunaclaw possesses within it a strength that you must use as one of the Claw.$B$BYou\'ve earned the right to use this Cenarion Lunardust, $N... now show me that you\'ve earned the right to possess the strength of body and the strength of heart needed as a druid!', 'Face Lunaclaw and earn the strength of body and heart it possesses.', 'Return to Turak Runetotem in Thunder Bluff.', 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', '', '', 52237);
+-- Update table quest_request_items
+DELETE FROM `quest_request_items` WHERE (`ID` = 6002);
+INSERT INTO `quest_request_items` (`ID`, `EmoteOnComplete`, `EmoteOnIncomplete`, `CompletionText`, `VerifiedBuild`) VALUES
+(6002, 1, 0, 'When you have faced the challenge that lies before you, your understanding of strength of body and strength of heart will be fully realized.  Until that time, I cannot help you further.', 52237);
+-- Update POi
+DELETE FROM `quest_poi` WHERE `QuestID`=6002;
+INSERT INTO `quest_poi` (`QuestID`, `id`, `ObjectiveIndex`, `MapID`, `WorldMapAreaId`, `Floor`, `Priority`, `Flags`, `VerifiedBuild`) VALUES 
+(6002, 0, -1, 1, 1639, 0, 0, 3, 52237),
+(6002, 1, 16, 1, 1156, 0, 0, 1, 52237);
+-- Update Poi Points
+DELETE FROM `quest_poi_points` WHERE `QuestID`=6002;
+INSERT INTO `quest_poi_points` (`QuestID`, `Idx1`, `Idx2`, `X`, `Y`, `VerifiedBuild`) VALUES 
```

Comment:
The values here aren't changing between latest master/this PR. Put the VerifiedBuild adjustments into an update.

Reply:
> The values here aren't changing between latest master/this PR. Put the VerifiedBuild adjustments into an update.

I put version 52237 because it was the most recent version that obtained these values, even though they are equal to 12340. If you don't need to change this, just go back to 12340, but they are equal to 52237 (current from blizzard)

Reply:
My point is don't do a delete/insert, but rather an update query affecting only the VerifiedBuild column since that's all that's changing.